core/src/cluster_info.rs

```diff
@@ -1616,50 +1616,48 @@ impl ClusterInfo {
             .unwrap()
     }
 
+    pub fn gossip_contact_info(id: &Pubkey, gossip_addr: SocketAddr) -> ContactInfo {
```

Comment:
Looks like this can be private:
```suggestion
   fn gossip_contact_info(id: &Pubkey, gossip_addr: SocketAddr) -> ContactInfo {
```